RestAPI_Automation/Requests/APIHelper.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Newtonsoft.Json;
+using RestSharp;
+
+namespace RestAPI_Automation.Requests;
+
+public class APIHelper<T>
+{
+    public string baseURL = "https://reqres.in/ ";
+
+    // concatenating base url and endpoint
+    public string SetUrl(string endpoint)
+    {
+        string url = Path.Combine(baseURL, endpoint);
+        return url;
```

Comment:
you can return Path.Combine(baseURL, endpoint);via labmda expression directly

Reply:
corrected